### PR TITLE
Fix a couple more warnings

### PR DESCRIPTION
**Pull request by raboof**

### CODE REVIEW

File: akka-http-tests/src/multi-jvm/scala/akka/remote/testkit/MultiNodeConfig.scala [hunk 1]
```diff
@@ -402,11 +402,11 @@ abstract class MultiNodeSpec(val myself: RoleName, _system: ActorSystem, _roles:
 
   // now add deployments, if so desired
 
-  private final case class Replacement(tag: String, role: RoleName) {
+  private final class Replacement(val tag: String, val role: RoleName) {
```
Comment (jrudolph) [inline akka-http-tests/src/multi-jvm/scala/akka/remote/testkit/MultiNodeConfig.scala:405]: Or just remove final?

**Review by jrudolph — APPROVED** [review]
LGTM, thanks

File: akka-http-tests/src/multi-jvm/scala/akka/remote/testkit/MultiNodeConfig.scala [hunk 2]
```diff
@@ -402,11 +402,11 @@ abstract class MultiNodeSpec(val myself: RoleName, _system: ActorSystem, _roles:
 
   // now add deployments, if so desired
 
-  private final case class Replacement(tag: String, role: RoleName) {
+  private case class Replacement(tag: String, role: RoleName) {
     lazy val addr = node(role).address.toString
   }
 
-  private val replacements = roles map (r => Replacement("@" + r.name + "@", r))
+  private val replacements = roles map (r => new Replacement("@" + r.name + "@", r))
```
Comment (jrudolph) [inline akka-http-tests/src/multi-jvm/scala/akka/remote/testkit/MultiNodeConfig.scala:409]:
```suggestion
  private val replacements = roles map (r => Replacement("@" + r.name + "@", r))
```

Comment (raboof) [reply akka-http-tests/src/multi-jvm/scala/akka/remote/testkit/MultiNodeConfig.scala]: Ups :+1:

**Pull request merged**